src/controlCenter/widgets/mpris/mpris.vala

```diff
@@ -174,6 +184,7 @@ namespace SwayNotificationCenter.Widgets.Mpris {
             string[] names = dbus_iface.list_names ();
             foreach (string name in names) {
                 if (!name.has_prefix (MPRIS_PREFIX)) continue;
+                if (is_blacklisted(name)) continue;
                 if (check_player_exists (name)) return;
```

Comment:
You may change this line to fix the unintended return

Reply:
Maybe I'm too ignorant about it, I'm seeing that my report was merged in this PR, but I don't know how to use info written here to fix the issue with my Swaync. I uninstalled the one I got from Arch repo, and compiled this one, still having the same issue: Swaync shows MPRIS module when nothing is being played, and when there's something being played, I have two different MPRIS modules in Swaync. What I am missing?

Thanks in advance.

Reply:
> Maybe I'm too ignorant about it, I'm seeing that my report was merged in this PR, but I don't know how to use info written here to fix the issue with my Swaync. I uninstalled the one I got from Arch repo, and compiled this one, still having the same issue: Swaync shows MPRIS module when nothing is being played, and when there's something being played, I have two different MPRIS modules in Swaync. What I am missing?
> 
> Thanks in advance.

By "compiled this one" do you mean you've cloned SwayNotificationCenter repo, pulled the commits to your repo and then built that?
Or are you manually patching the source?

I get the feeling you may simply be building `master` from this repo without the changes.

Reply:
Yeah, just cloned and built it. No commits pulled. I don't know how to pull the commits separately, I tried to just copy/paste the changes but apparently I did it wrongly, and I wasn't able to modify the installation...

Sorry for the ignorance :(